auto_max_lair.py

```diff
@@ -725,21 +724,25 @@ def select_pokemon(ctrlr) -> str:
     # Update statistics and reset stored information about the complete run.
     ctrlr.wins += 1 if run.lives != 0 else 0
     ctrlr.runs += 1
+    # calculate the win percent
```

Comment:
I think one of the issues before is that this update was done after any Discord message was sent, resulting in the count being 1 behind. For example, if a shiny was found on run 1, it would report 0/0 wins/runs since these numbers weren't calculated yet. I'm not sure this problem is fixed. Am I missing something?

Reply:
Yes the issue was that. For update messages, it was fine (because sent after it starts the run). But the shiny ping, matching stats, etc doesn't because it was sending the message before the end of the run

Reply:
Let's fix this small thing in a later update.